src/components/DescriptionList/_description-list.scss

```diff
@@ -0,0 +1,34 @@
+.cc-description-list__title {
+  display: block;
+  float: left;
+  font-size: var(--body-lg);
+  font-weight: bold;
+
+  margin: 0 var(--space-sm)  0 0;
```

Comment:
Remove (one of) the spaces after the margin right value (there is a double space)

Reply:
I feel like prettier **should** catch this, but it doesn't, which is annoying